config.go

```diff
@@ -95,12 +95,13 @@ func (c menderConfig) GetDeploymentLogLocation() string {
 	return c.UpdateLogPath
 }
 
-func (c menderConfig) GetCustomTenantToken() []byte {
-	return []byte(c.TenantToken)
-}
-
-func (c menderConfig) GetDefaultTenantTokenFileName() string {
-	return defaultTenantTokenFileName
+// GetTenantToken returns a default tenant-token if
+// no custom token is set in local.conf
+func (c menderConfig) GetTenantToken() []byte {
+	if c.TenantToken != "" {
+		return []byte(c.TenantToken)
+	}
+	return []byte(defaultTenantToken)
```

Comment:
@kacf now returns the token depending upon whether or not a token is set in local.conf. k?